lib/bomb_placer.ex

```diff
@@ -1,13 +1,14 @@
 defmodule BombPlacer do
-  @total_bomb_count 10
-
-  def place_bombs(board, randomizer, current_bomb_count \\ 0)
+  def place_bombs(board, randomizer, expected_bomb_count \\ 10) do
+    place_bombs_recursively(board, randomizer, expected_bomb_count, 0)
+  end
 
-  def place_bombs(board, _, @total_bomb_count) do
+  defp place_bombs_recursively(board, _, expected_bomb_count, bomb_count)
```

Comment:
Naming nit: it's a bit strange to call a recursive function `x_recursively`. This can just be named `place_bombs`, the guard clause is documentation!

Reply:
Thank you, @Thomascountz! Fixed!

Reply:
@Thomascountz could you approve this? (If you're happy with it, of course!)